S2VX.Game/Play/UserInterface/ScoreProcessor.cs

```diff
@@ -27,6 +28,30 @@ public class ScoreProcessor : CompositeDrawable {
         public S2VXSample Hit { get; private set; }
         public S2VXSample Miss { get; private set; }
 
+        public int PerfectCount { get; private set; }
+        public int EarlyCount { get; private set; }
+        public int LateCount { get; private set; }
+        public int MissCount { get; private set; }
+        public SortedList<double> Scores { get; private set; } = new SortedList<double>();
+        public int Combo { get; private set; }
+        public int MaxCombo { get; private set; }
+
+        public double Accuracy() => (double)PerfectCount / Scores.Count;
```

Comment:
May be more appropriate to use an Expression-Bodied Member(https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members) here by removing the parentheses since there's no setting being done, so a property with only a getter would make sense.

Reply:
Good to know. I like the shorter syntax, so I changed to using it. However, it kind of messes with my general conception of a property. I tend of think of properties as just direct values that you return, versus functions (with the parentheses), as something that calculates a result.

Reply:
Pog